ops/cicd-runner-build.cloudbuild.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+steps:
+  # Docker Build 
+  # Copying the client-libs for Docker context
+  - name: gcr.io/cloud-builders/docker
+    entrypoint: /bin/bash
+    dir: ops/cicd
```

Comment:
instead of calling this cicd, can we call this e2e-test or similar? We're no longer doing CI or CD as part of this process.

Reply:
`s/cicd/e2e/`

(IMO the "test" here is implied, and the shorter name is easier to understand at a glance.)